### PR TITLE
[Proposal] Add --skip-drop option

### PR DESCRIPTION
**Pull request by touyu**
issue: https://github.com/k0kubun/sqldef/issues/43

I implemented the `--skip-drop` option as one way to avoid destructive changes.
Please consider it.

### CODE REVIEW

File: cmd/mysqldef/mysqldef.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func parseOptions(args []string) (adapter.Config, *sqldef.Options) {
 		File     string `long:"file" description:"Read schema SQL from the file, rather than stdin" value-name:"sql_file" default:"-"`
 		DryRun   bool   `long:"dry-run" description:"Don't run DDLs but just show them"`
 		Export   bool   `long:"export" description:"Just dump the current schema to stdout"`
+		Safety   bool   `long:"safety" description:"Not make destructive changes such as Drop"`
```
Comment (k0kubun) [inline cmd/mysqldef/mysqldef.go]: I feel `--skip-drop` might be more descriptive about what it does than `--safety`. WDYT?

File: adapter/database.go [hunk 2]
```diff
@@ -42,13 +42,17 @@ func DumpDDLs(d Database) (string, error) {
...
-func RunDDLs(d Database, ddls []string) error {
+func RunDDLs(d Database, ddls []string, isSafety bool) error {
 	transaction, err := d.DB().Begin()
 	if err != nil {
 		return err
 	}
 	fmt.Println("-- Apply --")
 	for _, ddl := range ddls {
+		if isSafety && strings.Contains(ddl, "DROP") {
+			fmt.Printf("Not executed: %s;\n", ddl)
```
Comment (k0kubun) [inline adapter/database.go]: sqldef's output is designed to always be a valid input for `mysql`/`psql`. Could you prefix `--` to make it a SQL comment?

And if you follow https://github.com/k0kubun/sqldef/pull/44/files#r339372757, it'd be `-- Skipped: %s;`.

**Comment by k0kubun** [comment]
Could you add the same option to psqldef for consistency, and update the README as well?

Also, it'd be really nice if we can have tests in mysqldef_test.go and psqldef_test.go.

File: cmd/mysqldef/mysqldef.go (hunk 1 above)
Comment (touyu) [reply cmd/mysqldef/mysqldef.go]: I also think `--skip-drop` is better than `--safety` !

File: adapter/database.go (hunk 2 above)
Comment (touyu) [reply adapter/database.go]: I understand.
I think `-Skipped:% s;` is good.

**Comment by touyu** [comment]
@k0kubun 
Thank you for reviews.
I would be happy if you could check it again.

**Pull request merged**